### PR TITLE
[Fix] issue in notifications

### PR DESCRIPTION
**Pull request by jonasheinrich**
With and without my changes, I locally I got 8 errors in the attachment_controller_test.rb. Which I did not change. But Travis says everything is ok so it seems to be ok. 

Fixes https://www.openproject.org/work_packages/3980

### CODE REVIEW

File: db/migrate/20140127134733_fix_issue_in_notifications.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#++
+
+class FixIssueInNotifications < ActiveRecord::Migration
+  REPLACED = {
+    "issue_added" => "work_package_added",
+    "issue_updated" => "work_package_updated",
+    "issue_priority_updated" => "work_package_priority_updated",
+  }
+  def up
+    Setting['notified_events']= replace(Setting['notified_events'], REPLACED)
```
Comment (hschink) [inline db/migrate/20140127134733_fix_issue_in_notifications.rb]: There is a space missing: `Setting['notified_events']=` should become `Setting['notified_events'] =`

**Comment by hschink** [comment]
@jonasheinrich I only get emails for work packages I created but not for other changes. Would you mind to have a look on this issue? Thanks a lot!

File: config/locales/en.yml [hunk 2]
```diff
@@ -1317,7 +1320,7 @@ en:
   text_enumeration_destroy_question: "%{count} objects are assigned to this value."
   text_file_repository_writable: "Attachments directory writable"
   text_git_repo_example: "a bare and local repository (e.g. /gitrepo, c:\\gitrepo)"
-  text_work_package_added: "Work package %{id} has been reported by %{author}."
+  text_git_repo_example: "Work package %{id} has been reported by %{author}."
   text_work_package_category_destroy_assignments: "Remove category assignments"
```
Comment (marutosi) [inline config/locales/en.yml:1324]: ???

Comment (hschink) [inline config/locales/en.yml:1324]: @marutosi thanks for the hint! I missed that one. We are going to fix that soon.

Comment (jonasheinrich) [inline config/locales/en.yml:1324]: @marutosi  thanks again! here is the pull request to fix that: https://github.com/opf/openproject/pull/899

**Pull request merged**